CanHazFunny/CanHazFunny/OutputService.cs

```
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+    public class OutputService : IOutPutJoke
+    {
+        public void Print(string Joke)
```

Would we want to throw an exception if null here? (I'm not actually sure.)

---

Its not a bad idea. I think it would depend on how its used but I added it anyway. Its good to check for those things!